### ISSUE TITLE
[ED][Media - Concepts] Additional WCAG SC

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-media-intro#105]
I think that 1.2.7 should be added to the bottom list of SCs covered by tutuorials

> 1.2.7 Extended Audio Description (Prerecorded): Where pauses in foreground audio are insufficient to allow audio descriptions to convey the sense of the video, extended audio description is provided for all prerecorded video content in synchronized media. (Level AAA)

This is included on the Audio Descriptions page, so why not include it in the overview page.

**Comment by shawna-slh** [comment]
From [e-mail to wai-eo-editors](https://lists.w3.org/Archives/Public/wai-eo-editors/2017Sep/0013.html):

In the "Related WCAG 2.0 Resources" box on the overview page, you should also include SC 1.2.7 Extended Audio Description (Prerecorded). This is becoming a very viable alternative to 1.2.8 (and also meets Level AAA), and I see that you mention extended AD in your description decision tree.

**Comment by shawna-slh** [comment]
Also related: https://github.com/w3c/wai-media-intro/issues/99
